examples/titanic_example/run_titanic.py

```diff
@@ -168,12 +168,20 @@
 # - a Dockerfile on which the user can specify the required dependencies of the Python scripts
 
 inputs_metrics = [
-    FunctionInputSpec(identifier="datasamples", kind=AssetKind.data_sample, optional=False, multiple=True),
-    FunctionInputSpec(identifier="opener", kind=AssetKind.data_manager, optional=False, multiple=False),
-    FunctionInputSpec(identifier="predictions", kind=AssetKind.model, optional=False, multiple=False),
+    FunctionInputSpec(
+        identifier="datasamples", kind=AssetKind.data_sample, optional=False, multiple=True
+    ),
+    FunctionInputSpec(
+        identifier="opener", kind=AssetKind.data_manager, optional=False, multiple=False
+    ),
+    FunctionInputSpec(
+        identifier="predictions", kind=AssetKind.model, optional=False, multiple=False
+    ),
 ]
 
-outputs_metrics = [FunctionOutputSpec(identifier="performance", kind=AssetKind.performance, multiple=False)]
+outputs_metrics = [
+    FunctionOutputSpec(identifier="performance", kind=AssetKind.performance, multiple=False)
+]
```

Comment:
Is there something I can do, to not have theses formatting changes?

Reply:
do you have a settings.json in a .vscode folder ? And if yes, do you have smthg like 
`  "python.formatting.provider": "black",
  "python.formatting.blackArgs": ["--line-length", "120"],`